crud/select/filters.lua

```diff
@@ -251,29 +257,36 @@ local function add_collation_postfix(func_name, value_opts)
     error('Unsupported collation: ' .. tostring(value_opts.collation))
 end
 
-local function format_eq(cond)
-    local cond_strings = {}
-    local values_opts = cond.values_opts or {}
+local function get_lt_function_name_by_type(value_type, value_opts)
+    local func_name = 'lt'
 
-    for j = 1, #cond.values do
-        local fieldno = cond.fieldnos[j]
-        local value = cond.values[j]
-        local value_opts = values_opts[j] or {}
+    if value_type == 'boolean' then
+        func_name = 'lt_boolean'
+    elseif value_type == 'string' then
+        func_name = add_collation_postfix('lt', value_opts)
+    elseif value_type == 'uuid' then
+        func_name = 'lt_uuid'
+    end
 
-        local func_name = 'eq'
-        func_name = add_collation_postfix(func_name, value_opts)
+    return add_strict_postfix(func_name, value_opts)
+end
 
+local function get_eq_function_name_by_type(value_type, value_opts)
+    local func_name = 'eq'
+
+    if value_type == 'string' then
+        func_name = add_collation_postfix('eq', value_opts)
         if collations.is_unicode(value_opts.collation) then
             func_name = add_strict_postfix(func_name, value_opts)
         end
-
-        table.insert(cond_strings, format_comp_with_value(fieldno, func_name, value))
+    elseif value_type == 'uuid' then
+        func_name = 'eq_uuid'
     end
 
-    return cond_strings
+    return func_name
 end
 
-local function format_lt(cond)
+local function format_func(func_name_getter, cond)
```

Comment:
I sure that this refactoring can be performed in a separated patch.

Reply:
THis refactoring is related to the two new functions: `lt_uuid` and `eq_uuid`

Reply:
Just add these functions to the existing file. Please keep patch simpler. And we will refactor in the next patch.

Reply:
Moved common code back to the existing functions